README.md

```diff
@@ -33,7 +39,7 @@ The current version is built for Mac OS 10.12 SDK (as of version 2.4.7). A compi
  - **Cam 5** `/atem/program/5`
  - **Cam 6** `/atem/program/6`
  - and so on...
-  
+
```

Comment:
I wonder if it's worth changing the example here across to the newer models with more inputs, since that's a common question that comes up (where `/atem/program/7` is input 7, not colour bars etc.

Reply:
It could be, what do you have in mind?  I'm not as familiar with the input numbering on newer models

Reply:
Some (not all) of the list from https://github.com/danielbuechele/atemOSC/issues/69 I think should cover it well